.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -1,6 +1,6 @@
-name: New Feature
-description: For any issue related to creating a new feature
-title: "[Area] - Short Description"
+name: Feature Request
+description: Suggest a new feature for the project
```

Comment:
Not sure when this would be used

Reply:
The name feals funky since its basically a new feature, if we are adding something new from investigation